protocol/x/clob/keeper/msg_server_cancel_orders.go

```diff
@@ -37,6 +37,7 @@ func (k msgServer) CancelOrder(
 func (k Keeper) HandleMsgCancelOrder(
 	ctx sdk.Context,
 	msg *types.MsgCancelOrder,
+	isInternalOrder bool,
```

Comment:
nit: isVaultOrder
unless there are other orders this applies to?

Reply:
This was the existing terminology that @tqin7 implemented in HandleMsgPlaceOrder so I am keeping it consistent here. Currently this only applies to vault orders but i believe that they wanted to keep this more generic to possibly include other types of orders in the future